src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -174,6 +175,7 @@ public FeedbackSessionAttributes updateFeedbackSession(FeedbackSessionAttributes
 
         FeedbackSessionAttributes[] newAttributesFinal = new FeedbackSessionAttributes[] { null };
         try {
+            FeedbackSessionsDb thisDb = this;
             ofy().transact(new VoidWork() {
```

Comment:
Just a question: noted that only this DB uses transaction to update. Is it OK for other DB not using the transaction?

Reply:
Yup. Transaction is slow and we can tolerate the risk.